releasenotes/releasenotes_V0.7.2.md

```diff
@@ -0,0 +1,101 @@
+# Release Notes 0.7.2
+
+Keptn 0.7.2 enhances the 0.7.1 release with great UX/UI improvements; especially for working with Quality Gates. This release introduces a new API endpoint for triggering a quality gate evaluation and the Bridge shows more details about the evaluation result.
+
+**The key announcements of Keptn 0.7.2**:
+
+:rocket: *Improved UX for Quality Gates*: This release introduces an `/evaluation` endpoint that brings the same user-experience to the API as used from the Keptn CLI. Hence, easily trigger a quality gate evaluation based on the parameters `start`, `end`, and `timeframe` for a specific `service` in a particular `project`/`stage`.
+
+:sparkles: *Focused UI enhancements for Quality Gates*: The evaluation tile that displays the quality gate evaluation result has been improved in various ways. First, the Heatmap in this tile now highlights the currently selected evaluation result. Second, it also displays the evaluation results that were used for comparisons giving the user more insights into the quality gate. 
+
+:tada: *Usage statistics for Keptn installation*: If you want to get more insights into your Keptn installation, feel free to deploy the [statistics-service](https://github.com/keptn-sandbox/statistics-service) from the Keptn sandbox. This service provides usage statistics based on events and service executions.
+
```

Comment:
we should add one more bullet point here regarding the spec change for evaluation-done events.

*Spec update*: sh.keptn.events.evaluation-done cloud-events now have a new field called comparedEvents - see github.com/keptn/spec/... for more details.